core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -457,6 +457,7 @@
 import static org.apache.calcite.sql.fun.SqlStdOperatorTable.LAST_DAY;
 import static org.apache.calcite.sql.fun.SqlStdOperatorTable.LAST_VALUE;
 import static org.apache.calcite.sql.fun.SqlStdOperatorTable.LEAD;
+import static org.apache.calcite.sql.fun.SqlStdOperatorTable.LEFTSHIFT;
```

Comment:
SHIFT_LEFT would be a better name, throughout. (Someday we’ll have SHIFT_RIGHT, possibly SHIFT_RIGHT_LOGICAL etc.)